modules/cf-core/src/models/free-balance.ts

```diff
@@ -221,12 +221,12 @@ export function createFreeBalance(
 
   return new AppInstance(
     /* participants */ sortedTopLevelKeys,
-    /* defaultTimeout */ freeBalanceTimeout,
+    /* defaultTimeout */ toBN(freeBalanceTimeout).toHexString(),
     /* appInterface */ getFreeBalanceAppInterface(coinBucketAddress),
     /* appSeqNo */ HARD_CODED_ASSUMPTIONS.appSequenceNumberForFreeBalance,
     /* latestState */ serializeFreeBalanceState(initialState),
     /* latestVersionNumber */ 0,
-    /* latestTimeout */ HARD_CODED_ASSUMPTIONS.freeBalanceInitialStateTimeout,
+    /* latestTimeout */ toBN(HARD_CODED_ASSUMPTIONS.freeBalanceInitialStateTimeout).toHexString(),
```

Comment:
I'm not sure if we should have state-specific timeouts in the free balance app instance.